src/collections/_documentation/platforms/javascript/react.md

```diff
@@ -50,10 +50,10 @@ class ExampleBoundary extends Component {
     }
 
     componentDidCatch(error, errorInfo) {
-      Sentry.withScope(scope => {
+      Sentry.withScope(function(scope) {
           scope.setExtras(errorInfo);
           const eventId = Sentry.captureException(error);
-          this.setState({eventId});
+          this.setState({eventId: eventId});
```

Comment:
`this` is something not a react component here, because arrow function is not used.